docs/administration/authentication-providers/okta-authentication.md

```diff
@@ -0,0 +1,156 @@
+
+   ![](/docs/images/okta-authentication/okta-new-app-integration.png "width=400")
+
+4. Enter an **Application Name** like Octopus Deploy and for the **Login redirect URIs** enter `https://octopus.example.com/api/users/authenticatedToken/Okta` replacing `https://octopus.example.com` with the public url of your Octopus server, and click the **Save** button.
+
+   ![](/docs/images/okta-authentication/okta-create-openid-integration.png "width=500")
+
+:::hint
+**Reply URLs are case-sensitive**
+Please take care when adding this URL! They are **case-sensitive** and can be sensitive to trailing **slash** characters
+:::
+
+:::hint
+**Not using SSL?**
+That's OK, you can use `http` if you do not have SSL enabled on your Octopus Server. Please beware of the security implications in accepting a security token over an insecure channel.
+:::
+
+5. You should see the **General Settings** for the app you have just created. For the **Allowed grant types** ensure that both **Implicit (Hybrid)** and **Allow ID Token with implicit grant type** are checked. Click the **Save** button to continue.
+
+   ![](/docs/images/okta-authentication/okta-general-settings.png "width=500")
+
+### OpenID Connect Settings {#Oktaauthentication-OpenIDConnectSettings}
+
+There are two values you will need from the Okta configuration to complete the Octopus configuration: the **Client ID** and **Issuer**. (The Client ID is also referred to as Audience.)
+
+Select the **Sign On** tab and scroll down to the **OpenID Connect ID Token** section. Take note of the **Issuer** and **Audience** as you will need both these values to configure your Octopus server.
+
+![](/docs/images/okta-authentication/okta-openid-token.png "width=500")
+
+### Assign App {#Oktaauthentication-AssignApp}
+
+Next you will need to assign your app to people or groups within your Okta directory.
+
+1. Select the **Assignments** tab and click the **Assign** button. You can assign your app to people, and to groups.
+
+   ![](/docs/images/okta-authentication/okta-assign-app.png "width=500")
+
+2. To assign the app to all users, you can simply assign the default **Everyone** group to the app, and click **Done**.
+
+   ![](/docs/images/okta-authentication/okta-assign-to-groups.png "width=500")
+
+## Configuring Octopus Deploy Server {#Oktaauthentication-ConfiguringOctopusDeployServer}
+
+:::hint
+There is currently no UI for configuring Octopus to use Okta - it must be configured from the command line.
+:::
+
+You will need the **Client ID** (aka **Audience**) and **Issuer** obtained from the Okta portal as described above.
+
+:::success
+Your **Client ID** should be a string value like `0a4bxxxxxxxxxxxx9yc3`
+Your **Issuer** should be a URL like `https://dev-xxxxxx.oktapreview.com`
+:::
+
+Once you have those values, run the following from a command prompt in the folder where you installed Octopus Server:
+
+```powershell
+Octopus.Server.exe configure --OktaIsEnabled=true --OktaIssuer=Issuer --OktaClientId=ClientID
```

Comment:
I like sticking a fake `PS >` prompt at the start
`PS > & .\Octopus.Server.exe configure --OktaIsEnabled=true --OktaIssuer=Issuer --OktaClientId=ClientID`